app/workers/facilities/pssg_download.rb

```diff
@@ -58,7 +58,7 @@ def round_band(band)
 
     def extract_polygon(rings)
       geojson = "{\"type\":\"Polygon\",\"coordinates\":#{rings}}"
-      RGeo::GeoJSON.decode(geojson)
+      RGeo::GeoJSON.decode(geojson, geo_factory: RGeo::Geographic.spherical_factory(srid: 4326, uses_lenient_assertions: true))
```

Comment:
Awesome! Huge improvement from 139s to 9s to decode. When profiling, we noticed that the RGeo functions were in the call stack ~80% of the time.

Reply:
:clap: Awesome win. !!!